src/loss/RegressionLossEvaluator.py

```diff
@@ -107,7 +107,7 @@ def get_mean_norm_error(output_tensor: torch.Tensor, label_tensor: torch.Tensor,
         reshaped_tensor = diffs.view(diffs.shape[0], diffs.shape[1], -1, vec_size)
 
         # Compute the norm over the last dimension
-        norms = torch.norm(reshaped_tensor, dim=3)
+        norms = torch.norm(reshaped_tensor[:,-1:,:,:], dim=3)
```

Comment:
For a behavior change like this, we should either update the tests, or do this in a function with a different name to reflect its different behavior.